frontend/src/reducers/rootReducer.ts

```diff
@@ -24,12 +23,13 @@ import filterSlice from './filterSlice';
 import agencyReducer from './agencyReducer';
 import agencyDetailReducer from './agencyDetailReducer';
 import parcelLayerDataSlice from './parcelLayerDataSlice';
+import { accessRequestsSlice } from 'store/slices/accessRequests';
 
 export const reducerObject = {
```

Comment:
Use the `configureStore({reducer: { name: reducer, name: reducer } })` pattern.  Remove the need to have a separate string file.